diff --git a/setup.py b/setup.py
index 3273370..4300559 100644
--- a/setup.py
+++ b/setup.py
@@ -396,7 +396,7 @@ else:
     netcdf_lib_version = str(netcdf_lib_version)
     print(f'using netcdf library version {netcdf_lib_version}')
 
-DEFINE_MACROS = [("NPY_NO_DEPRECATED_API", "NPY_1_7_API_VERSION")]
+DEFINE_MACROS = []
 netcdf4_src_root = osp.join(osp.join('src','netCDF4'), '_netCDF4')
 netcdf4_src_c = netcdf4_src_root + '.c'
 netcdf4_src_pyx = netcdf4_src_root + '.pyx'