--- netcdf4-1.7.4.orig/src/netCDF4/_netCDF4.pyx	2026-01-04 23:09:45.000000000 -0300
+++ netcdf4-1.7.4/src/netCDF4/_netCDF4.pyx	2026-06-22 14:31:25.313297991 -0300
@@ -5630,7 +5630,7 @@
             # create a view so shape in caller is not modified (issue 90)
             try: # if extra singleton dims, just reshape
                 data = data.view()
-                data.shape = tuple(datashape)
+                data = data.reshape(tuple(datashape))
             except ValueError: # otherwise broadcast
                 data = numpy.broadcast_to(data, datashape)
 
@@ -6816,8 +6816,7 @@
     if encoding in ['none','None','bytes']:
         b = numpy.array(tuple(a.tobytes()),'S1')
     elif encoding == 'ascii':
-        b = numpy.array(tuple(a.tobytes().decode(encoding)),dtype+'1')
-        b.shape = a.shape + (n_strlen,)
+        b = (numpy.array(tuple(a.tobytes().decode(encoding)),dtype+'1')).reshape(a.shape + (n_strlen,))
     else:
         if not a.ndim:
             a = numpy.array([a])
@@ -6852,8 +6851,7 @@
         a = numpy.array([bs[n1:n1+slen] for n1 in range(0,len(bs),slen)],'S'+repr(slen))
     else:
         a = numpy.array([bs[n1:n1+slen].decode(encoding) for n1 in range(0,len(bs),slen)],'U'+repr(slen))
-    a.shape = b.shape[:-1]
-    return a
+    return a.reshape(b.shape[:-1])
 
 class MFDataset(Dataset):
     """
